samples/cellular/modem_shell/src/sock/sock_shell.c

```diff
@@ -174,15 +174,16 @@ static struct option long_options[] = {
 	{ "rai_one_resp",   no_argument,       0, SOCK_SHELL_OPT_RAI_ONE_RESP },
 	{ "rai_ongoing",    no_argument,       0, SOCK_SHELL_OPT_RAI_ONGOING },
 	{ "rai_wait_more",  no_argument,       0, SOCK_SHELL_OPT_RAI_WAIT_MORE },
-	{ 0,                0,                 0, 0  }
+	{ "help",           no_argument,       0, 'h' },
+	{ 0,                0,                 0, 0   }
 };
 
 static void sock_print_usage(void)
 {
 	mosh_print_no_format(sock_usage_str);
 }
 
-int sock_shell(const struct shell *shell, size_t argc, char **argv)
+static int sock_shell(const struct shell *shell, size_t argc, char **argv)
```

Comment:
IMHO, the sock command has too long help, it's 100 lines long. I think this command should be split into subcommand with separate help texts, but not in this PR.

Reply:
I've been doing this into similar form as link module's subcommands: https://github.com/nrfconnect/sdk-nrf/pull/12848#issuecomment-1780530644